addons/web/static/tests/views/list_tests.js

```diff
@@ -6085,17 +6084,14 @@ QUnit.module('Views', {
         await testUtils.dom.click(list.$('.o_data_row:eq(0) .o_data_cell:eq(0)'));
         await testUtils.fields.editInput(list.$('.o_field_widget[name=foo]'), "");
 
-        assert.containsOnce($('body'),'.modal', "should have a modal");
+        assert.containsOnce($('body'),'.modal', "should have a modal (invalid fields)");
         await testUtils.dom.click($('.modal button.btn'));
 
         // edit a field
         await testUtils.dom.click(list.$('.o_data_row:eq(0) .o_data_cell:eq(0)'));
         await testUtils.fields.editInput(list.$('.o_field_widget[name=foo]'), "bar");
 
-        assert.containsOnce($('body'),'.modal', "should have a modal");
-        await testUtils.dom.click($('.modal button.btn.btn-primary'));
-
-
+        assert.containsNone($('body'),'.modal', "should not have a modal");
         assert.strictEqual(list.$('.o_data_row:eq(0) .o_data_cell').text(), "bar",
             "the first row should be updated");
 
```

Comment:
is there a test asserting that the selection is kept if the user clicks on cancel? If not, you should probably add one